sprockets/clients/redis/__init__.py

```diff
@@ -109,7 +109,7 @@ def set(self, key, value, ttl=None):
         LOGGER.debug('Setting Redis key "%s"', key)
         ttl = ttl or self.config.ttl
         connection = self._get_shard_connection(key)
-        connection.set(key, value, ex=ttl)
+        connection.setex(name=key, time=ttl, value=value)
```

Comment:
This does not appear to be supported in python though: https://redis-py.readthedocs.org/en/latest/#redis.StrictRedis.set